philo/Makefile

```diff
@@ -31,11 +31,11 @@ INCLUDES = -I$(INCLUDES_DIR)
 all: $(NAME)
 
 $(NAME): $(OBJS)
-	$(CC) $(CFLAGS) $< -o $@ $(INCLUDES)
+	$(CC) $(CFLAGS) -o $@ $^
 
 $(OBJS_DIR)/%.o: $(SRCS_DIR)/%.c
 	@mkdir -p $(dir $@)
-	$(CC) $(CFLAGS) -c $< -o $@ $(INCLUDES)
+	$(CC) $(CFLAGS) $(INCLUDES) -c $< -o $@
```

Comment:
> -c 
C source code that must be preprocessed.

> -o
Place the primary output in file file. This applies to whatever sort of output is being produced, whether it be an executable file, an object file, an assembler file or preprocessed C code.
If -o is not specified, the default is to put an executable file in a.out, the object file for source.suffix in source.o, its assembler file in source.s, a precompiled header file in source.suffix.gch, and all preprocessed C source on standard output.